pages/02.administer/10.install/install.fr.md

```diff
@@ -68,70 +68,70 @@ Sélectionnez le matériel sur lequel vous souhaitez installer YunoHost :
 [div class="flex-container"]
 
 [div class="flex-child hardware{%if virtualbox %} active{% endif %}"]
-[[figure caption="VirtualBox"]![](image://virtualbox.png?height=75)[/figure]](/install/hardware:virtualbox)
+[[figure caption="VirtualBox"]![][image://virtualbox.png?height=75](/figure)](/install/hardware:virtualbox)
 [/div]
 
 [div class="flex-child hardware{%if rpi34 or rpi012 %} active{% endif %}"]
-[[figure caption="Raspberry Pi"]![](image://raspberrypi.png?height=75)[/figure]](/install/hardware:rpi34)
+[[figure caption="Raspberry Pi"]![][image://raspberrypi.png?height=75](/figure)](/install/hardware:rpi34)
 [/div]
 
 [div class="flex-child hardware{%if arm_sup or (arm_unsup and not rpi012) or internetcube %} active{% endif %}"]
-[[figure caption="Carte ARM"]![](image://olinuxino.png?height=75)[/figure]](/install/hardware:arm)
+[[figure caption="Carte ARM"]![][image://olinuxino.png?height=75](/figure)](/install/hardware:arm)
 [/div]
 
 [div class="flex-child hardware{%if regular %} active{% endif %}"]
-[[figure caption="Ordinateur standard"]![](image://computer.png?height=75)[/figure]](/install/hardware:regular)
+[[figure caption="Ordinateur standard"]![][image://computer.png?height=75](/figure)](/install/hardware:regular)
 [/div]
 
 [div class="flex-child hardware{%if vps_debian or vps_ynh %} active{% endif %}"]
-[[figure caption="Serveur distant"]![](image://vps.png?height=75)[/figure]](/install/hardware:vps_debian)
+[[figure caption="Serveur distant"]![][image://vps.png?height=75](/figure)](/install/hardware:vps_debian)
```

Comment:
uuuuh, wut, are we sure about this ? this looks like some sort of `[figure]stuff[/figure]` syntax ? with `stuff` being `![](someimage)`

Reply:
hm yeah it returns this error 
https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md011.md

we can tell it to shut up with:
<!-- markdownlint-disable-file MD011 -->